src/main/kotlin/de/cyface/uploader/DefaultUploader.kt

```diff
@@ -197,7 +199,11 @@ class DefaultUploader(private val apiEndpoint: String) : Uploader {
     }
 
     override fun endpoint(): URL {
-        return URL(DefaultAuthenticator.returnUrlWithTrailingSlash(apiEndpoint) + "measurements")
+        return URL(returnUrlWithTrailingSlash(apiEndpoint) + "measurements")
+    }
+
+    override fun filesEndpoint(measurementId: Long): URL {
+        return URL(returnUrlWithTrailingSlash(apiEndpoint) + "measurements/$measurementId/files")
     }
```

Comment:
This might change slightly but the Collector Part will be done separately in the future.